sources/packages/api/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -78,3 +78,8 @@ export interface ECertUploadResult {
   generatedFile: string;
   uploadedRecords: number;
 }
+
+export interface ESDCFileResponseDTO {
```

Comment:
Does this interface need to be renamed to ESDCFileResponseAPIOutDTO ? Maybe some code comment on what it is used for will also be helpful.

Reply:
Actually I saw another class with the same name "ESDCFileResponseAPIOutDTO" I recommended above while this one was deleted from the various imports. Is it used anywhere ?

Reply:
this is the interface is used by the service layer. `ESDCFileResponseAPIOutDTO` is used by controller layer

Reply:
`ESDCFileResponseDTO` is used in `sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts`